scripts/run_docker_build.sh

```diff
@@ -11,6 +11,7 @@ channels:
  - conda-forge
  - defaults
 
+always_yes: true
```

Comment:
In an interactive environment (like my laptop) I totally agree with you.  In an automated environment like this, I **despise** having CI halt because I forgot to add `--yes` to my conda command